channels-rke2.yaml

```diff
@@ -1049,3 +1089,32 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1500
     featureVersions: *featureVersions-v1
+  - version: v1.25.7-rc1+rke2r1
```

Comment:
We should probably replace v1.25.6 with v1.25.7 as none of v1.25.x versions were actually released yet for Rancher.